src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp

```diff
@@ -27,13 +15,7 @@ bool ov::intel_cpu::ACLInterpolateExecutor::init(const InterpolateAttrs &interpo
     acl_coord = arm_compute::SamplingPolicy::TOP_LEFT;
     auto& out_shape = dstDescs[0]->getShape().getDims();
 
-    int index_h, index_w;
-    if (!getIndices(dstDescs[0], index_h, index_w)) {
-        DEBUG_LOG("ACL Interpolate unsupported layout: ", dstDescs[0]->serializeFormat());
-        return false;
-    }
-
-    if ((aclInterpolateAttrs.coordTransMode == InterpolateCoordTransMode::pytorch_half_pixel && out_shape[index_h] > 1 && out_shape[index_w] > 1) ||
+    if ((aclInterpolateAttrs.coordTransMode == InterpolateCoordTransMode::pytorch_half_pixel && out_shape[2] > 1 && out_shape[3] > 1) ||
```

Comment:
I would still recommend to keep variables `index_h` and `index_w` for better code readability.